src/rustllvm/RustWrapper.cpp

```diff
@@ -469,12 +474,17 @@ extern "C" LLVMRustMetadataRef LLVMRustDIBuilderCreateStructType(
     unsigned LineNumber,
     uint64_t SizeInBits,
     uint64_t AlignInBits,
-    unsigned Flags,
     LLVMRustMetadataRef DerivedFrom,
     LLVMRustMetadataRef Elements,
     unsigned RunTimeLang,
     LLVMRustMetadataRef VTableHolder,
     const char *UniqueId) {
+#if LLVM_VERSION_GE(4, 0)
+  auto Flags = DINode::FlagZero;
+#else
+  unsigned Flags = 0;
```

Comment:
Hmm, could you point me to where `FlagZero` is defined? I was under the impression that that value did *not* exist until the enum change. Previously, it was just hardcoded as `0`.